src/script/auth/main.tsx

```diff
@@ -85,4 +86,5 @@ async function runApp() {
 }
 
 enableLogging(Config.getConfig().FEATURE.ENABLE_DEBUG);
+initializeDataDog(Config.getConfig());
```

Comment:
How about moving this call inside of `enableLogging` (so that you don't have to put it in both `main.tsx` and `app.ts`)?

Reply:
and we can pass the full config to the `enableLogging` method